databricks_cli/pipelines/api.py

```diff
@@ -63,6 +63,12 @@ def get(self, pipeline_id, headers=None):
     def reset(self, pipeline_id, headers=None):
         self.client.reset(pipeline_id, headers)
 
+    def run(self, pipeline_id, headers=None):
+        self.client.run(pipeline_id, headers)
+
+    def stop(self, pipeline_id, headers=None):
```

Comment:
I know this is pretty basic, but we should still add tests for it in test_api.py.